melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/mottak/SedMottakService.java

```diff
@@ -123,6 +168,73 @@ private void opprettOgLagreIdentifiseringsoppgave(SedMottattHendelse sedMottattH
         log.info("Opprettet oppgave med id {}", oppgaveID);
     }
 
+    private DnummerRekvisisjonTilMellomlagring byggDnummerRekvisisjonTilMellomlagring(SedMottattHendelse sedMottattHendelse, SED sed, Person personFraSed, boolean pinSEDErFraLandSedKommerFra) {
```

Comment:
Jeg får lyst til å flytte mapping til en egen klasse (smakssak)

Reply:
God idé! Jeg har laget egen mapper og tester for det.